src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -653,9 +653,12 @@ export class NotebookModel extends Disposable implements INotebookModel {
 			let standardKernels = find(this._standardKernels, kernel => this._defaultKernel && kernel.displayName === this._defaultKernel.display_name);
 			let connectionProviderIds = standardKernels ? standardKernels.connectionProviderIds : undefined;
 			let providerFeatures = this._capabilitiesService.getCapabilities(profile.providerName);
-			if (connectionProviderIds.length > 0 && this._currentKernelAlias) {
+			if (connectionProviderIds.length > 0) {
```

Comment:
`connectionProviderIds?.length`

It's possible for it to be undefined 2 lines above